doc/reference/reference_lua/box_info/synchro.rst

```diff
@@ -43,102 +46,121 @@ box.info.synchro
-                 > replication_synchro_quorum=2,
-                 > replication_synchro_timeout=1000
-                 > }
+        replication:
+          synchro_quorum: 2
+          synchro_timeout: 1000
 
-    Next, check the current state of synchronous replication:
+    Check the current state of synchronous replication:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> box.info.synchro
+        app:instance001> box.info.synchro
         ---
         - queue:
             owner: 1
+            confirm_lag: 0
             term: 2
+            age: 0
             len: 0
             busy: false
           quorum: 2
         ...
 
-    Create a space called ``sync`` and enable synchronous replication on this space.
-    Then, create an index.
+    Create a space called ``sync`` and enable synchronous replication on this space:
+
+    ..  code-block:: console
+
+        app:instance001> s = box.schema.space.create("sync", {is_sync=true})
+        ---
+        ...
+
+    Then, create an index:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> s = box.schema.space.create("sync", {is_sync=true})
-        tarantool> _ = s:create_index('pk')
+        app:instance001> _ = s:create_index('pk')
+        ---
+        ...
 
     After that, use ``box.ctl.promote()`` function to claim a queue:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> box.ctl.promote()
+        app:instance001> box.ctl.promote()
 
     Next, perform data manipulations:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> require('fiber').new(function() box.space.sync:replace{1} end)
+        app:instance001> require('fiber').new(function() box.space.sync:replace{1} end)
         ---
         - status: suspended
           name: lua
-          id: 119
+          id: 130
         ...
-        tarantool> require('fiber').new(function() box.space.sync:replace{1} end)
+        app:instance001> require('fiber').new(function() box.space.sync:replace{1} end)
         ---
         - status: suspended
           name: lua
-          id: 120
+          id: 131
         ...
-        tarantool> require('fiber').new(function() box.space.sync:replace{1} end)
+        app:instance001> require('fiber').new(function() box.space.sync:replace{1} end)
         ---
         - status: suspended
           name: lua
-          id: 121
+          id: 132
         ...
 
     If you call the ``box.info.synchro`` command again,
     you will see that now there are 3 transactions waiting in the queue:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> box.info.synchro
+        app:instance001> box.info.synchro
         ---
         - queue:
             owner: 1
+            confirm_lag: 0
             term: 2
-            len: 3
+            age: 0
+            len: 0
             busy: false
```

Comment:
@xuniq you can follow these steps to reproduce a non-empty queue:
```Lua
--------------------------------------------------------------------------------
--
-- Instance 1
--
-- Step 1
-- Create the first instance. It is going to be the leader.
box.cfg{
	listen = 3313,
	replication = {3313, 3314},
	replication_synchro_quorum = 2,
	replication_synchro_timeout = 1000,
}
box.schema.user.grant('guest', 'super')

--
-- Step 3
-- Create the sync space.
box.ctl.promote()
s = box.schema.space.create("sync", {is_sync=true})
_ = s:create_index('pk')

--
-- Step 5
-- Try to make some sync txns. They would hang, because the quorum is 2, and
-- the second instance is not available.
fiber = require('fiber')
for i = 1, 3 do fiber.new(function() box.space.sync:replace{i} end) end

--
-- Step 6
-- Observe the hanging transactions. Len == 3.
box.info.synchro

--------------------------------------------------------------------------------
--
-- Instance 2
--
-- Step 2
-- Create the second instance, it is a slave-instance. Just to elect the other
-- node as a leader.
box.cfg{
	listen = 3314,
	replication = {3313, 3314},
	read_only = true,
}
--
-- Step 4
-- Simulate failure like if this instance would crash or go out of the network.
os.exit(0)
```